dataactvalidator/config/sqlRules.csv

```diff
@@ -0,0 +1,4 @@
+rule_sql,rule_label,rule_description,rule_error_message,rule_cross_file_flag,file_type,severity_name
+"SELECT af.row, af.bydirectreimbursablefundingsource FROM award_financial AS af WHERE job_id = {} AND (ussgl480100_undeliveredordersobligationsunpaid_fyb IS NOT NULL OR ussgl480100_undeliveredordersobligationsunpaid_cpe IS NOT NULL OR ussgl483100_undeliveredordersobligtransferredunpaid_cpe IS NOT NULL OR ussgl488100_upadjsprioryearundelivordersobligunpaid_cpe IS NOT NULL OR ussgl490100_deliveredordersobligationsunpaid_fyb IS NOT NULL OR ussgl490100_deliveredordersobligationsunpaid_cpe IS NOT NULL OR ussgl493100_deliveredordersobligstransferredunpaid_cpe IS NOT NULL OR ussgl498100_upadjsprioryeardeliveredordersobligunpaid_cpe IS NOT NULL OR ussgl480200_undeliveredordersobligationsprepaidadvanced_fyb IS NOT NULL OR ussgl480200_undeliveredordersobligationsprepaidadv_cpe IS NOT NULL OR ussgl483200_undeliveredordersobligtransferredppdadv_cpe IS NOT NULL OR ussgl488200_upadjsprioryrundelivordersobligprepaidadv_cpe IS NOT NULL OR ussgl490200_deliveredordersobligationspaid_cpe IS NOT NULL OR ussgl490800_authorityoutlayednotyetdisbursed_fyb IS NOT NULL OR ussgl490800_authorityoutlayednotyetdisbursed_cpe IS NOT NULL OR ussgl498200_upadjsprioryrdelivordersobligpaid_cpe IS NOT NULL) AND COALESCE(bydirectreimbursablefundingsource,'') NOT IN ('R', 'D')",B12,"Reimbursable flag indicator is required when reporting obligation or outlay USSGL account balances (excluding downward adjustments SGL accounts). Valid values are ""R"" for ""ReimbursableFundingSource"" and ""D"" for ""DirectFundingSource"".","Reimbursable flag indicator is required when reporting obligation or outlay USSGL account balances (excluding downward adjustments SGL accounts). Valid values are ""R"" for ""ReimbursableFundingSource"" and ""D"" for ""DirectFundingSource"".",,award_financial,fatal
```

Comment:
For the cross file flag, is a blank entry the same thing as false?  Would it be better to put that in explicitly rather than interpret a blank entry?